src/applications/vaos/services/appointment/index.js

```diff
@@ -337,6 +337,39 @@ export function isUpcomingAppointmentOrRequest(appt) {
   );
 }
 
+/**
+ * Returns true if the given Appointment is a confirmed appointment
+ * or an Express Care request
+ *
+ * @export
+ * @param {Object} appt The FHIR Appointment to check
+ * @returns {Boolean} Whether or not the appointment is a valid upcoming
+ *  appointment or Express Care request
+ */
+export function isUpcomingAppointmentOrExpressCare(appt) {
+  if (CONFIRMED_APPOINTMENT_TYPES.has(appt.vaos.appointmentType)) {
+    const apptDateTime = moment(appt.start);
+
+    return (
+      !appt.vaos.isPastAppointment &&
+      !FUTURE_APPOINTMENTS_HIDDEN_SET.has(appt.description) &&
+      apptDateTime.isValid() &&
+      apptDateTime.isBefore(moment().add(13, 'months'))
+    );
+  }
+
+  return (
+    appt.vaos.isExpressCare &&
```

Comment:
EC logic is kind of guessing, I think we need to sit down with Peter and get a better understanding of how EC is handled in the refresh.